migrations/versions/18124bf31811_add_uprn_mapping_table.py

```diff
@@ -1,18 +1,18 @@
 """Add UPRN mapping table
```

Comment:
Need to ignore this update totally. Make sure we have the 'old' version 36316acef1a as it's the same as production.